### DISCUSSION TITLE
Is it ok if I rename the default user?

### DISCUSSION THREAD

**Discussion by eamonburns** [termux/termux-app#3674] [Q&A] [answered]
I've noticed the default user is called `u0_a476`, is there a particular reason for this? It also looks kind of ugly, would it break anything if I renamed it with `usermod -l super_awesome_and_cool_username u0_a476`?

**Comment by twaik** [comment, marked as answer]
You can not. Username is managed by android, it contains of user id (`u0`) and the Android application sandbox id (`a476` in your case). It can not be changed or masked.
